### ISSUE TITLE
Dashing is not working in Microsoft Edge in Windows 10

### ISSUE THREAD

**Issue by bombastic1** [Shopify/dashing#662]
Hi! I cannot open Dashing in Edge from Windows 10. Please consider adding support for a new modern browser.

**Comment by MartynKeigher** [comment]
Check out Issue #310. Might not give you ALL the answers, but might point you in the right direction of a workaround.

**Comment by bombastic1** [comment]
+ reply@- notifications@Thank you. We will try this workaround. 13.01.2016, 03:43, "Martyn Keigher" <notifications@github.com>:Check out Issue #310. Might not give you ALL the answers, but might point you in the right direction of a workaround.—Reply to this email directly or view it on GitHub.   -- Сергей Кульбачинскийhttp://staff.yandex-team.ru/bombastic/

**Comment by MartynKeigher** [comment]
Hey @bombastic1.

Is this still an issue? If so, let us know. if not... please mark this issue as CLOSED.

Thanks

://mk

**Comment by DeanCording** [comment]
This is still an issue. #310 doesn't appear to fix the problems when using Edge.

**Comment by bombastic1** [comment]
Yes, I confirm that even after patching (from #310) this is still an issue.

**Comment by mekliches** [comment]
still not working after more than a year
